x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/components/loading_state.tsx

```diff
@@ -11,12 +11,14 @@ import { EuiPageContent, EuiSpacer, EuiLoadingContent } from '@elastic/eui';
 
 import { SetAppSearchChrome as SetPageChrome } from '../../../../shared/kibana_chrome';
 
+import { ENGINES_TITLE } from './../constants';
+
 import { EnginesOverviewHeader } from './header';
 
 export const LoadingState: React.FC = () => {
   return (
     <>
-      <SetPageChrome />
+      <SetPageChrome trail={[ENGINES_TITLE]} />
       <EnginesOverviewHeader />
```

Comment:
[Optional] I'd actually suggest DRYing this out by putting `<SetPageChrome />`in `<EnginesOverviewHeader />`, since all 3 pages use this component.

Reply:
That's a great suggestion, thanks!
DRYed it out here: https://github.com/elastic/kibana/pull/96636/commits/95db518d56882b919addd79835a329ba40e7eee4

Reply:
Forgot to mention that I tested the new change and it works the same.